cloudbuild.yaml

```diff
@@ -9,14 +9,25 @@ steps:
   args: [
             'build',
             '-t', 'us.gcr.io/$PROJECT_ID/$REPO_NAME:$COMMIT_SHA',
+            '-t', 'getsentry/snuba:ci',
```

Comment:
Any reason why we are tagging this as `ci`? We should be able to pass `COMMIT_SHA` as a build arg and parametrize the `Dockerfile.testing` file as follows:

```dockerfile
ARG COMMIT_SHA
FROM getsentry/snuba:${COMMIT_SHA}

RUN pip install -r requirements-test.txt
```